source/_integrations/wake_on_lan.markdown

```diff
@@ -128,3 +128,21 @@ switch:
 shell_command:
   turn_off_TARGET: 'ssh hass@TARGET sudo pm-suspend'
 ```
+
+#### Shutdown Windows (via MQTT using IOT Link)
+
+- Install and Configure [IOT Link](https://iotlink.gitlab.io/)
+
+```yaml
+switch:
+  - platform: wake_on_lan
+    name: "TARGET"
+    ...
+    turn_off:
+      service: mqtt.publish
+      data:
+        topic: iotlink/workgroup/computername/commands/shutdown
+```
+**Note**: Replace `workgroup` and `computername`.
+
+See [IOT Link Wiki](https://gitlab.com/iotlink/iotlink/wikis) for further MQTT Commands like Suspend, Hibernate and much more. 
```

Comment:
This is a nice example, but not related to the wake_on_lan integration. This part belongs the documentation of IoT link.

Reply:
Hm, thats kind of sad, but it's you decision.

There is already one "real-life examples of how to use the turn_off variable" for Suspending Linux via SSH.

I don't quiet understand why this does not qualify as an other "real-life Example".

Reply:
Sure, I can make many more. There is one that is more universal for the platforms Home Assistant supports. Another example with natively provided integration of Home Assistant would be nice as well. 

This example, however, is completely external, which is not a concern of this project.